test_unit.py

```diff
@@ -745,7 +747,7 @@ def test_with_all_fields(
         }
         host_schema.return_value.load.return_value.data = host_input
 
-        result = deserialize_host(host_input)
+        result = deserialize_host(input, host_schema)
```

Comment:
Here again, the variable name is _host_input_. _input_ is a Python built-in. It works only because we are mocking the schema and it doesn’t 💥  on invalid type.

In the other tests, we are passing in just `{}` to make it evident that the input value is ignored. Doing that it’d be at least consistent.

```suggestion
        result = deserialize_host({}, host_schema)
```